keps/NNNN-kep-template/README.md

```diff
@@ -452,6 +453,36 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+### Testing Quality
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### What is the current (unit) test coverage of packages that will be touched when implementing this KEP?
+
+<!--
+List the packages where non-boilerplate code will be added/changed with their
+current coverage as reported by test-coverage suite under
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+-->
+
+###### If the coverage is below 80%, what additional tests will be added before implementing this KEP?
+
+<!--
+We need to ensure that all existing areas of the code that will be touched by
+this enhancement are adequatly tested to mitigate the risk of regressions.
+--->
+
+###### Please link the integration and e2e tests that were added/extended for this KEP
+
+<!--
+This question should be filled when targeting Beta release.
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+So even if those hit Alpha in such state, it won't be possible to target Beta
```

Comment:
IMO everything should always be tested. alpha/beta/GA doesn't matter for this requirement.

Reply:
(As a note, all KEPs are required to have a test plan regardless of stage, so +1 on lavalamps point above.)

Reply:
> IMO everything should always be tested. alpha/beta/GA doesn't matter for this requirement.

I agree with the principle. But the principle was always there but didn't work very well in practice.
So I would like have a way to ensure that we won't proceed to Beta if shortcuts were made in Alpha.

Reply:
If the principle is clearly there already, and it's not being followed, then there's likely no modification to this template that will fix that, since the problem is enforcement and not the instructions.

Reply:
> If the principle is clearly there already, and it's not being followed, then there's likely no modification to this template that will fix that, since the problem is enforcement and not the instructions.

I don't fully agree. Because I can also imagine that being something not fully conscious.
If you're forced to fill in (or review) the template here for Beta graduation, this will force you to think about this.